.goreleaser.yml

```diff
@@ -14,6 +14,65 @@ builds:
       - -buildmode=pie
     binary: syscalls-bumper
 
+dockers:
+  - use: buildx
+    goos: linux
+    goarch: amd64
+    dockerfile: Dockerfile
+    image_templates:
+      - "falcosecurity/syscalls-bumper:latest-amd64"
+      - "falcosecurity/syscalls-bumper:{{ .Version }}-amd64"
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+    extra_files:
```

Comment:
I think that `extra_files` is wrong; in syscalls-bumper, it should be just:
```
.git
go.mod
go.sum
main.go
Makefile
```
i think.